lib/auth/windows/windows.go

```diff
@@ -253,15 +299,21 @@ func SubjectAltNameExtension(user, domain string) (pkix.Extension, error) {
 // Types for ASN.1 SAN serialization.
 
 // SubjectAltName is a struct for marshaling the SAN field in a certificate
```

Comment:
Mention what the type parameter is for in the godoc.

Reply:
b6081bf6d6156405d70e2cf3d69bd74a4712b6f3